src/ui/box/chrome.jsx

```diff
@@ -309,7 +299,23 @@ export default class Chrome extends React.Component {
             </div>
           </MultisizeSlide>
         </div>
-        {submitButton}
+        {/*
```

Comment:
previous code wrapped the `<SubmitButton.../>` in between `{}`. Now you're just wrapping a comment and then adding the button. Is that OK?